service/config.go

```diff
@@ -0,0 +1,101 @@
+package service
+
+import (
+	"fmt"
+	"github.com/microdevs/missy/log"
+	"os"
+	"sync"
+)
+
+var config *Configuration
+var once sync.Once
+
+// Config returns the singleton  instance of the service configuration
+func Config() *Configuration {
+	once.Do(func() {
```

Comment:
Wouldn't it be cleaner to just do this?
```go
if config == nil {
	config = &Configuration{}
}
return config
```
I don't see any advantage of using a `sync.Once` here.

Reply:
http://marcio.io/2015/07/singleton-pattern-in-go/

Reply:
Ah, yeah. Makes sense.
But since we don't need lazy init here you could also move the initialization into `init()` and return `config` from here straight.

Reply:
because registering (missy) parameters happens in init() functions too, I wanted it with the first possible RegisterParameter() call to avoid race conditions

Reply:
Ah. Okay, makes sense to me now.
Can you put that as a comment so others can understand easily why an `init` func isn't used?